tests/networkpolicy_test.go

```diff
@@ -99,13 +99,13 @@ var _ = Describe("[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:compon
 			Expect(err).ToNot(HaveOccurred())
 		})
 
-		It("[test_id:CNV-1511] should be failed to reach vmia from vmib", func() {
+		It("[test_id:1511] should be failed to reach vmia from vmib", func() {
 			By("Connect vmia from vmib")
 			ip := vmia.Status.Interfaces[0].IP
 			assertPingFail(ip, vmib)
 		})
 
-		It("[test_id:CNV-1512] should be failed to reach vmib from vmia", func() {
+		It("[test_id:1512] should be failed to reach vmib from vmia", func() {
```

Comment:
The test does what is in the description, however the test steps says otherwise. @dshchedr would you please take a look?

For the reference: https://polarion.engineering.redhat.com/polarion/#/project/CNV/workitem?id=CNV-1512

Reply:
@duyanyan can you please take a look? as for me - it seems like mistake in polarion testcase steps

Reply:
Fixed.